rootlesskit.yaml

```diff
@@ -1,10 +1,15 @@
 package:
   name: rootlesskit
   version: 1.1.1
-  epoch: 0
+  epoch: 3
```

Comment:
nit, maybe change this back to 1? I assume this is because of local devl / changes? 🤣

Reply:
Done.